app/controllers/v1/tokens_controller.rb

```diff
@@ -0,0 +1,21 @@
+module V1
+  class TokensController < V1::BaseController
+    skip_before_action :authenticate_user!
+
+    def create
+      result = CreateJwt.call(authentication_params)
```

Comment:
Should we really introduce custom logic for generating a token? It is already provided by a gem. Also creating a separate JwtToken model not necessary. Looks like you are doing it only to response in JSON API format, but maybe we can return token here in JSON API manually? 

```ruby
class V1::TokensController < Knock::AuthTokenController
  skip_before_action :verify_authenticity_token, only: :create

  def create
    render json: response_data, status: :created
  end

  private

  def response_data
    { data: { id: auth_token.token, type: "tokens" } }
  end

  def entity_class
    User
  end

  def auth_params
    jsonapi_params(only: %i[email password])
  end
end
```
In this case, we can remove **JwtToken** model, **CreateJwt** interactor and **JwtTokenSerializer**. Any thoughts?